### ISSUE TITLE
Does instance normalisation support in current version of texture_nets?

### ISSUE THREAD

**Issue by crazylyf** [DmitryUlyanov/texture_nets#29]
Hi,
Does the current version texture_nets support _instance normalisation_? I could not find function like that in the project, instead, a BatchNormalisation found in the utils.lua.

Best

**Comment by DmitryUlyanov** [comment]
If you use `batch_norm` separately for each object it will be equal to instance norm. It is exactly what is happening in the code.

**Comment by jounih** [comment]
Would you be able to briefly describe how to use batch_norm separately for each object? thanks!

**Comment by DmitryUlyanov** [comment]
Look at 
https://github.com/DmitryUlyanov/texture_nets/blob/master/train.lua#L113

Instead of doing forward for `B x C x W x H` tensor do B times for `1 x C x W x H`

**Comment by crazylyf** [comment]
@DmitryUlyanov Greak! thanks a lot!
